code/modules/client/preference_setup/loadout/lists/suits.dm

```diff
@@ -84,6 +84,20 @@
 	)
 	gear_tweaks += new /datum/gear_tweak/path(longcoat)
 
+/datum/gear/suit/serbian_overcoat
+	display_name = "serbian overcoat"
+	path = /obj/item/clothing/suit/storage/serbian_overcoat
+	cost = 4 // decent armor values
```

Comment:
Risks creating a new meta-pick for the loadout, a better solution would be tweaking the serb crates to be more affordable, no?